src/wallet/spend.cpp

```diff
@@ -1392,4 +1392,384 @@ bool FundTransaction(CWallet& wallet, CMutableTransaction& tx, CAmount& nFeeRet,
 
     return true;
 }
+
+constexpr int NUM_DENIABILIZATION_OUTPUTS = 2;
```

Comment:
why is this hardwired to 2?

Reply:
2 outputs is a common spend transaction and we're trying to mimic that. 
Another concern with a larger output count is that the number of deniabilization chunks grows much larger with each "cycle", and thus becomes more expensive to perform further cycles.
In the end 2 looked like the sweet spot, but I made the constant just in case we want to play with other output counts in the future.
I'll add comments in the code to that effect so that it's more obvious.
Thanks!

Reply:
Maybe commenting the constant helps,
maybe best commented in some function,
at the extremely minimum least I think such intrincacies should be commented in the commit message introducing either the constant or whatever function